integration/subxt-tests/src/cases/asserts.rs

```diff
@@ -45,10 +45,7 @@ async fn case() -> anyhow::Result<()> {
             &|t: &ContractMessageTranscoder| t.encode::<_, String>("test_assert_rpc", []).unwrap(),
         )
         .await;
-
-    if let Err(r) = res {
-        assert!(r.to_string().contains("ContractTrapped"));
-    }
+    assert!(res.is_err());
```

Comment:
why not 
```rust
 assert!(res.unwrap_err().to_string().contains("ContractReverted"));
```
?

Reply:
I tried that but the error just contains `[]` somehow. However it works for the transaction just further down, where I do this assertion.

Reply:
So there is a bug, right?

Reply:
The way the execution is build for these tests, the error would contain the debug buffer output. However it's compiled with `--release` so there is no debug buffer output and that's why the error is empty. I'll change that.